library/core/src/fmt/mod.rs

```diff
@@ -444,8 +444,9 @@ impl<'a> Arguments<'a> {
     /// assert_eq!(format_args!("{}", 1).as_str(), None);
     /// ```
     #[stable(feature = "fmt_as_str", since = "1.52.0")]
+    #[rustc_const_unstable(feature = "const_arguments_as_str", issue = "none")]
```

Comment:
This should probably have a tracking issue. `issue = "none"` usually means that this will never be stable.